spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplateTests.java

```diff
@@ -35,6 +35,7 @@
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
+import org.springframework.cloud.gcp.pubsub.core.publisher.PubSubPublisherTemplate;
```

Comment:
You have changes this test-case, so add your name to the `@author` list as well.

Reply:
Added name to the `@author list`